src/nanoemoji/config.py

```diff
@@ -47,8 +47,10 @@
 
 # we use None as a sentinel for flag not set; FontConfig class has the actual defaults.
 # CLI flags override config file (which overrides default FontConfig).
-flags.DEFINE_string("config", None, "Config file")
+flags.DEFINE_string("config", None, "Config file.")
 flags.DEFINE_integer("upem", None, "Units per em.")
+flags.DEFINE_integer("version_major", None, "Major version.")
```

Comment:
```suggestion
flags.DEFINE_integer("version_major", 1, "Major version.")
```

Reply:
Actually, it's correct like this. The flags default should be all `None`, in this way the FontConfig default values will be used when a config parameter is not overridden in a config file nor with a CLI flag. Am I right @rsheeter?

Reply:
Correct, we take None as "no flag provided"; the default goes on the Config struct